src/kernel/MomentumMassElemKernel.C

```diff
@@ -160,7 +163,7 @@ MomentumMassElemKernel<AlgTraits>::execute(
     for (int ic=0; ic < AlgTraits::nodesPerElement_; ++ic) {
       const int icNdim = ic * AlgTraits::nDim_;
       const DoubleType r = v_shape_function_(ip, ic);
-      const DoubleType lhsfac = r * gamma1_ * rhoNp1 * scV / dt_;
+      const DoubleType lhsfac = r * gamma1_ * rhoNp1 * scV / dt_ * diagRelaxFactor_;
```

Comment:
Is this required? I see that the `diagRelaxFactor_` is 1.0.

Reply:
`diagRelaxFactor_` is determined by the relaxation factor provided in the input file by the user. It is defaulted to `1.0` but will not be always `1.0`. See Line 85 in this file.